packages/openneuro-components/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react'
+import './tooltip.scss'
+
+export interface TooltipProps {
+  tooltip: string
+  flow: 'up' | 'down' | 'left' | 'right'
+}
+
+export const Tooltip = ({
+  children,
```

Comment:
Need to include children in the `interface`. The type for `children` would be `React.ReactNode`